valeriano-manassero/trino/templates/deployment-coordinator.yaml

```diff
@@ -48,6 +48,14 @@ spec:
           secret:
             secretName: trino-password-authentication
         {{- end }}{{- end }}
+        {{- if (include "trino.tlsEncryption" . ) }}
+        - name: certs
+          secret:
+            secretName: {{ include "trino.tlsSecretName" . }}
+            defaultMode: 0256
```

Comment:
That was/is my impression, but 256 and 0400 would give me permission issues. I couldn't even read the keystore file I think it was? 0256 actually gives me the correct permissions. I guess I need to retest..

Reply:
That's with securityContext btw, but I'm not sure why securityContext would mess things up like this

Reply:
I'm not surprised. If that **is** being interpreted as octal, the `0256` would mean `write-only for the owner`, `read and execute for the group` and `read and write for everyone`. Because of that latter "everyone" permission, you can do whatever you want with that file. But I'm pretty sure this is a symptom of the owner / group being wrong... If you look at other places where `defaultMode` is used in our charts, you'll see a bunch of `0444` (used alongside the PV being marked as read-only). But if `fsGroup` is set correctly, you should only need `0440`. I don't believe `0400` can work if we're running as a non-root user - sorry, I should have caught that earlier.